src/main/java/com/seriouscompany/business/java/fizzbuzz/packagenamingpackage/impl/Constants.java

```diff
@@ -0,0 +1,19 @@
+package com.seriouscompany.business.java.fizzbuzz.packagenamingpackage.impl;
+
+public class Constants {
+
+	public static final String AN_ATTEMPT_WAS_MADE_TO_DIVIDE_BY_ZERO = "An attempt was made to divide by zero.";
```

Comment:
make sure to send an email to all of our mailing lists and to each developers' managers